src/Stratis.Bitcoin/Connection/ConnectionManager.cs

```diff
@@ -211,14 +209,14 @@ private void StartNodeServer()
 
         public void AddDiscoveredNodesRequirement(NetworkPeerServices services)
         {
-            this.discoveredNodeRequiredService |= services;
```

Comment:
`discoveredNodeRequiredService` was never used actually this is odd.
So what is the default services we start the node with?
It looks like `Nothing` is set to be the default which is probably wrong.